modules/govuk/manifests/apps/content_performance_manager.pp

```diff
@@ -76,6 +77,10 @@
     app    => $app_name,
   }
 
+  govuk::procfile::worker {'content-performance-manager':
```

Comment:
You could use `$app_name` here instead of `content-performance-manager` which is what the manifests for some other apps do.